tests/ConnectionTest.php

```diff
@@ -231,4 +233,29 @@ public function testDriverName()
         $driver = DB::connection('mongodb')->getDriverName();
         $this->assertEquals('mongodb', $driver);
     }
+
+    public function testPingMethod()
+    {
+        $config = [
+            'name'     => 'mongodb',
+            'driver'   => 'mongodb',
+            'dsn'      => env('MONGODB_URI', 'mongodb://mongodb/'),
```

Comment:
To match the default configuration in other tests (see `tests/config/database.php`):
```suggestion
            'dsn'      => env('MONGODB_URI', 'mongodb://127.0.0.1/'),
```